test/Dialect/FIRRTL/errors.mlir

```diff
@@ -1948,3 +1948,18 @@ firrtl.circuit "RWProbeInstance" {
     %rw = firrtl.ref.rwprobe <@RWProbeInstance::@inst> : !firrtl.uint<1>
   }
 }
+
+// -----
+
+firrtl.circuit "ModulePortClassMissing" {
```

Comment:
A bit more tests would be nice, maybe something re:type not matching (`verifyType`), and at least one entry showing its non-error usage :+1: .